Mathlib/RingTheory/Localization/Ideal.lean

```diff
@@ -167,14 +170,14 @@ variable {R : Type _} [CommRing R] (M : Submonoid R) (S : Type _) [CommRing S]
 
 variable [Algebra R S] [IsLocalization M S]
 
-include M
+set_option maxHeartbeats 400000 in
```

Comment:
Could you please leave a porting note to explain why this is needed?
If you add `set_option synthInstance.etaExperiment true`, do you still need to increase the heartbeats?

Reply:
I have just tried adding `set_option synthInstance.etaExperiment true` but I still need to increase the heartbeats afterwards. Indeed, there is a timeout at line 203 (I do not understand why and I was about to ask on Zulip). Meanwhile, I have written a porting note to indicate that there was a timeout and where that timeout was.